packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -226,14 +226,7 @@ const Header: React.FC<Props> = ({
           {t('skip_to_content')}
         </a>
         <div className={styles.menu}>
-          <IconButton
-            className={styles.iconButton}
-            aria-label={sideBarOpen ? t('close_menu') : t('open_menu')}
```

Comment:
Do you think we can remove this condition?

Reply:
Yes, we think so: The `aria-expanded` toggle already indicates whether or not it is “Open menu”, just like we currently do with the “View password” label on the view/hide password toggle.